swap/src/cli/command.rs

```diff
@@ -90,3 +111,11 @@ fn parse_btc(str: &str) -> Result<bitcoin::Amount> {
     let amount = bitcoin::Amount::from_str_in(str, ::bitcoin::Denomination::Bitcoin)?;
     Ok(amount)
 }
+
+fn parse_multiaddr(str: &str) -> Multiaddr {
+    str.parse().expect("default alice multiaddr is valid")
```

Comment:
I think the `expect` message `"default alice multiaddr is valid"` is incorrect. `parse_multiaddr` is not only used in the default case but also when an address is provided by the user. This should be a better error message!

`structopt` is able to handle `anyhow::Result` - have a look at how we parse the Bitcoin amount - I think this should be similar and the actual error should be yielded if the parsing fails.

Reply:
fixed by using default parse from_str